fix(scim): remove beta tag for SCIM for roles (#6129)
getsentry/sentry-docs: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx

## Known Issues

> - The Import Users feature is not currently supported because Sentry's SCIM API doesn't support the User `name` attribute fields: `firstName` and `lastName`at this time. For compatibility purposes, they're returned with `N/A` values.

Reviewer: ```suggestion
- The Import Users feature isn't currently supported because Sentry's SCIM API doesn't support the user `name` attribute fields: `firstName` and `lastName` at this time. For compatibility purposes, they're returned as `N/A` values.
```
